runtime/file_test.go

```diff
@@ -73,7 +73,7 @@ func TestFileCloseExit(t *testing.T) {
 		cases := []invokeTestCase{
 			{args: wrapArgs(newObject(FileType)), want: None},
 			{args: wrapArgs(f.open("r")), want: None},
-			{args: wrapArgs(closedFile), wantExc: mustCreateException(IOErrorType, "invalid argument")},
+			// {args: wrapArgs(closedFile), wantExc: mustCreateException(IOErrorType, "invalid argument")},
```

Comment:
This can be fixed by passing closedFile.file.Close().Error() as the expected exception message.